src/dodal/devices/shutter.py

```diff
@@ -0,0 +1,23 @@
+from enum import Enum
+
+from ophyd_async.core import StandardReadable
+from ophyd_async.epics.signal import epics_signal_rw
+
+
+class OpenState(str, Enum):
+    CLOSED = "closed"
+    OPEN = "open"
+    OPENING = "opening"
+    CLOSING = "closing"
+    FAULT = "fault"
+
+
+class Shutter(StandardReadable):
+    def __init__(self, prefix: str, name: str):
+        self.position = epics_signal_rw(
+            write_pv=prefix + "CTRL2", read_pv=prefix + "STA", datatype=OpenState
+        )
+        super().__init__(
+            name=name,
+        )
+        self.set_readable_signals([self.position])
```

Comment:
nit: Slightly neater maybe, but I won't hold the change up anymore.

```suggestion
        with self.add_children_as_readables(ConfigSignal):
            self.position = epics_signal_rw(
                write_pv=prefix + "CTRL2", read_pv=prefix + "STA", datatype=OpenState
           )
        super().__init__(name=name)
```

Reply:
I had no idea this method exists on the StandardReadable. what difference does it make?

if it's just another way to do the same thing ie syntactic sugar, it's not Pythonic
Zen of Python line 13
> There should be one-- and preferably only one --obvious way to do it.

Reply:
It was added during the bluesky hackathon (i.e. "very" recently) and shows up in the auto-generated docs for the StandardReadable, but I agree that it isn't obvious which is why it's a nit.

https://github.com/bluesky/ophyd-async/pull/240/files

Reply:
done.